README.md

```diff
@@ -200,16 +200,21 @@ Check solve section for steganography.
 - [CSWSH](http://ironwasp.org/cswsh.html) - Cross-Site WebSocket Hijacking Tester
 - [Request Bin](http://requestb.in/) - Lets you inspect http requests to a particular url
 
-## Stegano
+## Steganography
 
 *Tools used for solving Steganography challenges*
 
 - [Convert](http://www.imagemagick.org/script/convert.php) - Convert images b/w formats and apply filters
+- [exif](http://manpages.ubuntu.com/manpages/trusty/man1/exif.1.html) - shows EXIF information in JPEG files
+- [exiftool](https://linux.die.net/man/1/exiftool) - Read and write meta information in files
+- [exiv2](http://www.exiv2.org/manpage.html) - Image metadata manipulation tool
 - [ImageMagick](http://www.imagemagick.org/script/index.php) - Tool for manipulating images
+- [outguess](https://www.freebsd.org/cgi/man.cgi?query=outguess+&apropos=0&sektion=0&manpath=FreeBSD+Ports+5.1-RELEASE&format=html) - universal steganographic tool
```

Comment:
As a general rule for maintaining convention and standard, first letter of description should be captial.